rzup/src/toolchain/repo.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::utils::{fetch_release_info, get_http_client, GithubReleaseData, Repo};
+use anyhow::Result;
+use clap::{Parser, ValueEnum};
+
+#[derive(Default, Debug, Clone, Parser, ValueEnum)]
+pub enum ToolchainRepo {
+    #[default]
+    Rust,
+    Cpp,
```

Comment:
@SchmErik brought up a really good point. If I want rust support, I will also want the c++ toolchain since I'll want to allow C code that is called from rust code to work out of the box. Same goes for future golang support. So C is really like a 'base' feature.

Reply:
Sounds good, Erik and I landed on the same page about this and it will be installed by default as well.